srml/support/src/storage/storage_items.rs

```diff
@@ -50,109 +50,127 @@ pub use crate::rstd::marker::PhantomData;
 #[doc(hidden)]
 pub use crate::rstd::boxed::Box;
 
+#[doc(hidden)]
+pub fn id<T>(t: T) -> T {
+	t
+}
+
+#[doc(hidden)]
+pub use Some;
```

Comment:
Isn't `Some` always in the prelude? Even in `no_std`?

Reply:
yes true. it is only because we define the keyword as an ident and expand later in the macro with `$crate::storage::storage_items::$into_query(v)`

at some point I could actually change ident to path and put full path earlier